src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const listItems = document.querySelectorAll('li');
+
+for (const item of listItems) {
+  const textContent = item.childNodes[0];
```

Comment:
The code assumes that the first child of the list item is a text node. If there are other nodes (like comments or whitespace text nodes) before the actual text node, this could lead to unexpected behavior. Consider using `item.firstChild` or iterating over `item.childNodes` to find the first text node.